development/request_guidelines.md

```diff
@@ -1,3 +1,21 @@
+# Pull Request Expectations
+
+We have a set of norms around pull requests and merging of code. Most of these have evolved in an informal way among development teams or during our retrospectives while discussing things we could improve. There are exceptions to many rules, but this document attempts to identify our baseline expectations.
+
+## For Submitters
+
+- When containers are modified as a part of a pull request manifests should be updated to use those containers. Alternatively, instructions may be provided to inform reviewers how to deploy the change.
```

Comment:
nit, comma after pull request

Reply:
We might want to rephrase this a bit, since ddev doesn't really have manifests, but the same situation still applies there. Possible rephrase, feel free to use or adjust as interested:

"When a pull request involves updating a container image, a provisional tag for that feature should be pushed to the drud org. The pull request against the project referencing this container should include the provisional tag. This helps to ensure automated testing is run against the updated container, and ensures reviewers are testing with the updated asset as well."